### ISSUE TITLE
Marci's new reformulated skill Sidekick doesn't work as described

### ISSUE THREAD

**Issue by xnescaux** [ValveSoftware/Dota2-Gameplay#10530]
### Ability name

Sidekick

### Description

When using the skill does not provide additional damage as described in the description, only the lifesteal

### Example Match ID (and possibly Timestamp)

7277942556 - 09/08/2023 - 19:32

**Comment by TheChetu** [comment]
Working as intended. You can see from your timestamp Slark has 129+18 Damage before Sidekick and goes up to 129+66 after Sidekick is applied, similarly Marci goes from 131+21 to 131+69 Damage.
When watching replays you need to be cautious as going back and forth over a time period can cause discrepancies, best to watch from a point before a buff/debuff is applied and watch fully through to avoid any desyncs.

**Comment by xnescaux** [comment]
indeed, but have other problem, in the skill description, when holding "ALT", it says that it can be used on itself providing half the bonus, but this cannot be done.

**Comment by TheChetu** [comment]
> 

The additional/alternative text you are referring to attached below is:
Marci can self-cast Sidekick to buff herself and the nearest allied hero within 700 range. Lifesteal effectiveness gets a 40% penalty against creeps.

This says nothing about providing half the bonus if self-cast. The half the bonus portion is if Marci is outside of the buffed ally range (over 2500 range). However the alt-text is wrong in that self-casting does not work, likely an oversight to prevent self-casting with no allies nearby functioning but still broken functionality as it does not work with an ally nearby.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/9092744/e21429d8-3267-4599-8b94-8519765eda40)

**Comment by TheChetu** [comment]
Here is a related issue with Sidekick though, if you apply it to an ally and level the skill without reapplying the ability the visual effect of the bonus damage increases but the damage does not change until reapplied. Can be seen in the following screenshots:

Level 1 Sidekick Applied
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/9092744/fd271341-3062-4eee-9cc8-41835d51a32b)

Level 2 Sidekick Leveled but not Re-applied (Doing Level 1 Damage)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/9092744/0a7ed0f0-9bdf-4610-b579-06fc68845618)

Level 3 Sidekick Leveled but not Re-applied (Doing Level 1 Damage)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/9092744/e995ad0d-4779-480b-a185-2e299a631cf6)

Level 4 Sidekick Leveled but not Re-applied (Doing Level 1 Damage)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/9092744/602f101c-221e-4470-8a6d-957401601d33)

Level 4 Sidekick Leveled and Re-applied (Doing Level 4 Damage)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/9092744/19145a68-0271-4797-831c-aac37af1b790)
